src/core/cache.ts

```diff
@@ -131,12 +135,21 @@ export async function getAllContracts(
       weth: new ethers.Contract(await ethWethEarn.asset(), erc20Abi, provider),
       router: Router__factory.connect(ethRouter.address, provider),
     };
+  } else if (chainId == 8453 || chainId == 84531) {
+    const baseUsdEarn = VaultV2__factory.connect(baseUsdEarnData.address, provider);
```

Comment:
We should handle the cases where `baseUsdEarnData` might be `undefined`. For example - for testnets, it might be undefined

Reply:
Done